app/_hub/appsentinels/appsentinels/overview/_index.md

```diff
@@ -6,24 +6,22 @@ At the platform's core is an AI/ML engine, AI Sentinels, which combines multiple
 
 ## How it works
 
-The AppSentinels plugin performs logging and enforcement (blocking) of API transactions. The plugin seamlessly integrates with Kong to provide for visibility and protection.
+The AppSentinels plugin performs logging and enforcement (blocking) of API transactions. The plugin seamlessly integrates with {{site.base-gateway}} to provide visibility and protection.
 
-AppSentinels plugin works in 2 modes,
-1. Logging or transparent mode
-A copy of the request and response transactions is made and asynchronously shared with AppSentinels Edge Controller to provide for visibility and security. Integrations can help provide for enforcement like blocking of bad IPs and threat actors.
+The AppSentinels plugin works in the following two modes:
+1. **Logging or transparent mode**: A copy of the request and response transactions is made and asynchronously shared with AppSentinels Edge Controller to provide for visibility and security. Integrations can help provide for enforcement like blocking of bad IPs and threat actors.
 
-2. Enforcement mode
-This mode provides for transaction level blocking. Incoming requests are held until a verdict is given by the AppSentinels Edge Controller. A negative response of enforcement from the latter will result in the request being dropped from further processing. In case of higher latency of a verdict, the plugin performs a fail open to ensure business continuity.
+2. **Enforcement mode**: This mode provides for transaction level blocking. Incoming requests are held until the AppSentinels Edge Controller provides a verdict. If the Controller provides a negative enforcement response of enforcement, the request is dropped from further processing. In case of higher latency of a verdict, the plugin performs a fail open to ensure business continuity.
```

Comment:
```suggestion
2. **Enforcement mode**: This mode provides transaction level blocking. Incoming requests are held until the AppSentinels Edge Controller provides a verdict. If the Controller provides a negative enforcement response of enforcement, the request is dropped from further processing. In case of higher latency of a verdict, the plugin performs a fail open to ensure business continuity.
```